docs/extensions/postgres_scanner.md

```diff
@@ -4,4 +4,49 @@ title: Postgres Scanner
 selected: Documentation/Postgres Scanner
 ---
 
-See [the repo](https://github.com/duckdblabs/postgresscanner#usage) for usage, or the [official announcement](../../2022/09/30/postgres-scanner) for implementation details and background.

```

Comment:
Probably worth keeping the announcement link for the implementation background?

Reply:
Thanks, done